plugins/linux-v4l2/v4l2-helpers.h

```diff
@@ -91,24 +94,24 @@ static inline enum video_format v4l2_to_obs_video_format(uint_fast32_t format)
  * and the height in the low word.
  * The array is terminated with a zero.
  */
-static const int v4l2_framesizes[] = {
+static const uint64_t v4l2_framesizes[] = {
```

Comment:
Why is this a `uint64_t` instead of `int64_t` as elsewhere in packed numbers?

Reply:
Fixed (in a second), it should be an int64_t.